launcher/tests/trusted-extensions.spec.ts

```diff
@@ -46,6 +46,13 @@ const PRODUCT_JSON_WITH_EXTENSIONS_ALTERNATIVE = `{
 		]
 	}
 }`;
+const PRODUCT_JSON_TWO_EXTENSIONS_UPPERCASE = `{
```

Comment:
Cold you remove this variable at all and use `PRODUCT_JSON_TWO_EXTENSIONS` in the test below?
I poster my thoughts about that to the original issue.